### ISSUE TITLE
custom deployment folder

### ISSUE THREAD

**Issue by kgoedert** [redhat-developer/vscode-rsp-ui#103]
Hello,

is there a way to configure a custom deployment folder for a wildfly server?

**Comment by lstocchi** [comment]
Hello @kgoedert , have you tried by changing the standalone.xml file?
You should add something like (after the extensions block):
```
<paths>
        <path name="deploy.dir" path="/new/deployment/path/here"/>
</paths>
```
then change the deployment scanner to look at that path

```
<deployment-scanner path="deployment" relative-to="deploy.dir" scan-interval="5000" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
```
Unfortunately i don't think we can automate everything right now so you still have to add the war you want to deploy manually to the path you specified above.

**Comment by robstryker** [comment]
I'll look into this.

**Comment by kgoedert** [comment]
Sorry for the delayed response... just trying to clarify my specific need. 

Your solution is not ideal for me because during development I use a wildfly inside a docker container. For deployments, I usually use a shared folder, which is where I deploy my war file, which is mapped to a docker volume.

When I saw the description of this feature, my impression was that I could use visual code to copy my war into this shared folder. Without any other modification. I was looking for some configuration like you have in eclipse jboss tools, where you configure your deployment folder and then you can use the full publish option.

**Comment by robstryker** [comment]
So to be clear, you're not actually using the server adapter in vscode to start or stop the wfly or the docker container it's used in? You're just looking for a server adapter which can deploy to an arbitrary folder? Is that about right?

**Comment by kgoedert** [comment]
I can start/stop the server, using the docker extension. So initially, yes, the deploy to the arbitrary folder would be enough.

**Comment by robstryker** [comment]
Sorry for the extra question, but...  Do you require the .dodeploy marker files to be created? Or do you just need to be able to copy the files in? Trying to determine the best course here.

**Comment by kgoedert** [comment]
The .dodeploy file would be nice

**Comment by robstryker** [comment]
Seems like this was fixed in the rsp-server repository:

* acf44326 - Fixes #531 - custom deploy folder for wildfly (2 months ago) <Rob Stryker>
https://github.com/redhat-developer/rsp-server/commit/acf44326d8db50489ad6fd41c0df070e67e4676a

You can overwrite the deploy directory by adding the following property to your server. (Right click your server, edit it in editor, should present a json file with various properties etc).  Then you can add the property key wildfly.server.deploy.directory   and use either an absolute filesystem path or a path relative to your server home directory as the value. 

Let me know if this is fixed for you in the latest issue.
